iwyu_test_util.py

```diff
@@ -404,7 +416,7 @@ def TestIwyuOnRelativeFile(test_case, cc_file, cpp_files_to_check,
 
   # TODO(csilvers): verify that has exit-status 0.
   cmd = '%s %s %s %s' % (
-      _IWYU_PATH, ' '.join(iwyu_flags), ' '.join(clang_flags), cc_file)
+      _GetIwyuPath(), ' '.join(iwyu_flags), ' '.join(clang_flags), cc_file)
```

Comment:
Sorry, I've missed that `_GetIwyuPath` caches its result. Performance concern was a false alarm.